code/obj/mining_cloud_storage.dm

```diff
@@ -329,7 +352,17 @@
 
 		return src.loc
 
+	get_desc(dist, mob/user)
+		. = ..()
+
+		if (src.status & BROKEN)
+			. += "It looks broken and inoperable."
+
+
 	ui_interact(mob/user, datum/tgui/ui)
+		if (src.status & BROKEN)
+			return
```

Comment:
no feedback?

Reply:
Have added some more messages on all the interactions to tell the user why its not working.